ingestor/inputs/vector_embeddings.py

```diff
@@ -90,8 +100,13 @@ async def extract_data(collection: Any, page_size: int, page_offset: int):
         "indexTerms.earliestageorloweststage": 1,
         "indexTerms.datecollected": 1
     }
+
+
+    # query = {"_id": {"$gt": ObjectId("66882105ffdc56ce50b5f5f2") }}
+    query = {"_id": {"$gt": ObjectId("66882305ffdc56ce50b64652") }}
```

Comment:
Ignore.  (Added to avoid redundant effort after various errors have caused the worker to crash).